lib/statsample-glm/glm/base.rb

```diff
@@ -28,7 +28,20 @@ def initialize ds, y, opts={}
                             .const_get("#{algorithm}").const_get("#{method}")
                             .new(@data_set, @dependent, @opts)
       end
-      
+
+      # Returns the coefficients of trained model
+      #
+      # @param [Symbol] as_a Specifies the form of output
+      #
+      # @return [Array, Hash] coefficients of the model
```

Comment:
In this and some other methods it looks like, if `as_a=:array` is specified, then a `Daru::Vector` is returned, rather than an `Array`... Is that an issue? (If it is, I think that it can be a separate pull request though)